### ISSUE TITLE
Learning progress tracking for individuals and groups.

### ISSUE THREAD

**Issue by gravitz** [UstadMobile/UstadMobile#722]
The ability for teachers to track and report on the learning progress of students (individuals and groups). This requirement can be traced back to the inception report.

**Comment by mikedawson** [comment]
I don't think this was in the inception report. There is support in assignments for submission of assignments by individuals and groups. Searching for the word group in the inception report doesn't bring this up.

Beyond assignment group submission, I haven't ever heard or seen any use case brought up from government partners or users for this.
